target/linux/ramips/mt7620/base-files/etc/hotplug.d/firmware/10-rt2x00-eeprom

```diff
@@ -40,6 +40,9 @@ case "$FIRMWARE" in
 		caldata_extract "factory" 0x0 0x200
 		caldata_patch_mac $wifi_mac 0x4
 		;;
+	edimax,ew-7476hpc-16m)
+		rt2x00_eeprom_extract "factory" 0 512
```

Comment:
Do we still have this function? Size should be in hex notation.

Reply:
Dropped